prody/proteins/interactions.py

```diff
@@ -2087,6 +2203,80 @@ def calcDistribution(interactions, residue1, residue2=None, **kwargs):
                 LOGGER.info(i)
 
 
+def saveInteractionsAsDummyAtoms(atoms, interactions, filename, **kwargs):
+    '''Creates a PDB file which will contain protein structure and dummy atoms that will be placed between pairs
+    of interacting residues.
+    
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+    
+    :arg interactions: list of interactions
+    :type interactions: list
+
+    :arg filename: name of the PDB file which will contain dummy atoms and protein structure
+    :type filename: str 
+    
+    :arg RESNAME_dummy: resname of the dummy atom, use 3-letter name
+                        be default is 'DUM'
+    :type RESNAME_dummy: str '''
+
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+                            
+    RESNAME_dummy = kwargs.pop('RESNAME_dummy', 'DUM')
+    
+    def putDUMatom(coord1, coord2):
+        midpoint = [
+            (coord1[0] + coord2[0]) / 2,
+            (coord1[1] + coord2[1]) / 2,
+            (coord1[2] + coord2[2]) / 2
+        ]
+        return midpoint
+
+    all_DUMs = []
+    atoms_ = atoms.copy()
+
+    for i in interactions:
+        if len(i[1].split('_')) <= 3:
+            res1_name = 'chain '+i[2]+' resname '+i[0][:3]+' and resid '+i[0][3:]+' and index '+' '.join(i[1].split('_')[1:])            
```

Comment:
I'm surprising we don't need an `and` before `resname` in this line and all the ones below. We don't if it works but it could be easier to read